src/api/gnosisProtocol/api.ts

```diff
@@ -200,3 +200,27 @@ export async function getProfileData(chainId: ChainId, address: string): Promise
     return response.json()
   }
 }
+
+const NETWORK_TO_API_PREFIX = {
+  [ChainId.MAINNET]: 'mainnet',
+  [ChainId.GOERLI]: 'goerli',
+  [ChainId.GNOSIS_CHAIN]: 'xdai',
+}
+
+export type TotalSurplusData = {
+  totalSurplus: string
+}
+
+// TODO: Move to the SDK
+export async function getSurplusData(chainId: ChainId, address: string): Promise<TotalSurplusData> {
+  console.log(`[api:${API_NAME}] Get surplus data for`, chainId, address)
+
+  const baseUrl = `https://barn.api.cow.fi/${NETWORK_TO_API_PREFIX[chainId]}/api`
```

Comment:
Should we use `api.cow.fi` for prod?

Reply:
Not currently available.
Will work on that once backend has it ready